### PR TITLE
Draft: Add options to start and stop group replication

### PR DESCRIPTION
**Pull request by eryx12o45**
##### SUMMARY
- Added options to start and stop group replication

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
mysql_replication moodule

### CODE REVIEW

**Comment by Andersson007** [comment]
@eryx12o45 hi, thanks for the PR. It's a draft but anyway, please
1) add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to)
2) add tests to make CI green
3) maybe it's worth adding underscores to the new values to make them more human friendly?

Thanks!

**Comment by eryx12o45** [comment]
@Andersson007 thanks for the comments. I've added a changelog fragment and renamed the commands. Unfortunately I will not be able to provide tests in the near future, because of time constraints and the fact, that I would need to extend the setup routine to add group replication configs, etc. otherwise the commands would never work :-(

File: changelogs/fragments/mysql_replication_add_group_replication_commands.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - mysql_replication - added options to `start_group_replication` and `stop_group_replication`
```
Comment (Andersson007) [inline changelogs/fragments/mysql_replication_add_group_replication_commands.yml:2]:
```suggestion
  - mysql_replication - add ``start_group_replication`` and ``stop_group_replication`` options (https://github.com/ansible-collections/community.mysql/pull/93).
```

**Review by Andersson007 — COMMENTED** [review]
@eryx12o45 thanks for fixing! I'll take a deeper look when time allows

**Comment by Andersson007** [comment]
@betanummeric @laurent-indermuehle @bmalynovytch @rsicart @Jorge-Rodriguez what do you think about these changes? Can we merge it w/o tests? (provided that the fragment is fixed)

**Comment by Jorge-Rodriguez** [comment]
This PR needs rebase. I would wait for #63 to be merged and then rework these changes on top of it. Then we could add tests as well.

**Comment by Andersson007** [comment]
@eryx12o45 could you please [rebase](https://docs.ansible.com/ansible/latest/dev_guide/developing_rebasing.html) the PR? if you have no time, let us know

**Comment by eryx12o45** [comment]
New version here --> https://github.com/ansible-collections/community.mysql/pull/647

**Pull request closed without merging**